overlord/snapstate/snapstate.go

```diff
@@ -204,9 +204,14 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup) (*state.T
 	}
 
 	installSet := state.NewTaskSet(tasks...)
-	configSet := Configure(st, snapsup.Name(), defaults)
-	configSet.WaitAll(installSet)
-	installSet.AddAll(configSet)
+
+	// gross hack, do not run configure hook on classic (LP: #1668738)
+	// for now until we understand why it is failing for some people
+	if !(release.OnClassic && snapsup.Name() == "core") {
```

Comment:
For this case we can ignore ubuntu-core, it does not (and never had) a configure hook.